gbfile.cc

```diff
@@ -84,7 +85,13 @@ gzapi_open(gbfile* self, const char* mode)
     SET_BINARY_MODE(fd);
     self->handle.gz = gzdopen(fileno(fd), openmode);
   } else {
+#if __WIN32__
+    // On Windows, use gzopen_w() to handle Unicode paths.
+    std::wstring wname = QString(self->name).toStdWString();
+    self->handle.gz = gzopen_w(wname.c_str(), openmode);
+#else
```

Comment:
if you remove the qPrintable at in gbfile.cc at old line 542 you will need to add one at new line 93 like you did in util.cc at new line 274.

Reply:
Thanks, fixed (I think) below.

Reply:
Also worth noting that with Qt 5.7 and later we can use qUtf16Printable() to convert straight from a QString() to a temporary const wchar_t*.